docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Metrics.adoc

```diff
@@ -0,0 +1,105 @@
+[[MicroProfile_Metrics_SmallRye]]
+= Metrics Subsystem Configuration
+
+[NOTE]
+====
+This subsystem exposes only base metrics from the WildFly Management Model and JVM MBeans.
+
+Support for https://microprofile.io/project/eclipse/microprofile-metrics[Eclipse MicroProfile Metrics] is provided by
+the _microprofile-metrics-smallrye_ subsystem.
+====
+
+[[required-extension-metrics]]
+== Extension
+
+This `org.wildfly.extension.metrics` extension is included in all the standalone configurations included in the
+WildFly distribution as well as the `metrics` layer.
+
+You can also add the extension to a configuration without it either by adding
+an `<extension module="org.wildfly.extension.metrics"/>`
+element to the xml or by using the following CLI operation:
+
+[source,options="nowrap"]
+----
+[standalone@localhost:9990 /] /extension=org.wildfly.extension.metrics:add
+----
+
+== Management Model
+
+The `/subsystem=metrics` resource defines three attributes:
+
+* `security-enabled` - a boolean to indicate whether authentication is required to access the HTTP metrics endpoint (described below). By default, it is `true`. The
+standalone configurations explicitly sets it to `false` to accept unauthenticated access to the HTTP endpoints.
+* `exposed-subsystems` - a list of strings corresponding to the names of subsystems that exposes their metrics in the HTTP metrics endpoints.
+By default, it is not defined (there will be no metrics exposed by subsystem. The special wildcard `*` can be used to expose metrics from _all_ subsystems. The standalone
+configuration sets this attribute to `*`.
```

Comment:
Same. Not rendered properly.

Reply:
fixed (this seems to be an issue with Asciidoc rendering